BPCheck/Check_BP_Servers.sql

```diff
@@ -13434,6 +13434,39 @@ BEGIN
 		END;
 	END;
 
+--------------------------------------------------------------------------------------------------------------------------------
+---
+---
+--### Indexes with fill factor = 100 pct subsection
+-- - You can set @ptochecks to OFF in this block if you want to skip more performance tuning and optimization oriented checks.
+--------------------------------------------------------------------------------------------------------------------------------
+	IF @ptochecks = 1
+	BEGIN
+		RAISERROR (N'  |-Starting Indexes with fill factor = 100 pct', 10, 1) WITH NOWAIT
+		IF (SELECT COUNT(*)
```

Comment:
What about?
`IF EXISTS (SELECT TOP 1 1`

Reply:
Good point. I just reused the code that is currently in the stored procedure. Yes in some situations it could exists big differences in performance but in this case is just metadata information and not thousands of rows so differences in performance are not expected.